tools/cloudera-parcel-gcsconnector/create_parcel.sh

```diff
@@ -177,7 +177,11 @@ function main() {
     graceful_exit
   fi
 
-  cp * ${PARCEL_FULLNAME}/lib/hadoop/lib/
+  # Copy GCS connector jar
+  cp gcs-connector-hadoop2-latest.jar ${PARCEL_FULLNAME}/lib/hadoop/lib/
+  
+  # Copy service account key file
+  cp *.json ${PARCEL_FULLNAME}/lib/hadoop/lib/
```

Comment:
@yhqs540 sorry for the delay -- is this a good enough fix?

Reply:
looks good. Thanks